keps/693-multikueue/README.md

```diff
@@ -316,6 +328,67 @@ that ware created by this multikueue manager cluster and delete them if their lo
 - `WorkerLostTimeout` - defines the time a local workload's multikueue admission check state is kept Ready
 if the connection with its reserving worker cluster is lost.
 
+
+### Since Kueue 0.13: MultiKueue Dispatcher API
+
+In order to meet the requirements of [Story 3](#story-3) we introduce an API for custom dispatching algorithms.
```

Comment:
```suggestion
Since Kueue 0.13, in order to meet the requirements of [Story 3](#story-3), we introduce an API for custom dispatching algorithms.
```